assets/src/design-system/components/contextMenu/menuItem.js

```diff
@@ -120,7 +108,7 @@ export const MenuItem = ({
       : {};
 
     return (
-      <StyledLink
+      <Link
```

Comment:
No longer do we show the blue border when clicking the context menu. Design said it was too much